assets/src/scss/styleguide.scss

```diff
@@ -177,17 +177,17 @@ code {
 // p.typeSpecimen::after  { content: "Font stack: #{ko-font(base)}"; }
 
 .sg-typeSpecimen--body {
-	font-size: ko-rem(30);
+	font-size: 30px;
```

Comment:
More a question than a comment – what’s the reason we are using pixels over the font-size mixin in the styleguide?